strategy.md

```diff
@@ -35,3 +35,28 @@ Working with management on what they perceive to be preventing them from accepti
 ## Community Building & Engagement
 
 We will also focus on keeping the community engaged and involved in the various activities through social media and advertising methods.
+
+## Session Dates
```

Comment:
I think it might be useful to track this on a separate page.

Reply:
Created a separate page for the dates and added a link from the strategy page.